test/hooks.test.js

```diff
@@ -2727,6 +2727,27 @@ test('onRegister hook should be called / 4', t => {
   })
 })
 
+test('early termination, onRequest', t => {
```

Comment:
Is a test for early termination with the signature `(request, reply, done)` already in place?

Reply:
Yes https://github.com/fastify/fastify/blob/c052c21db4b1d8b80eafd3435a6ba4ccdc9f97f8/test/hooks.test.js#L1213